provider/source/src/units/helpers.rs

```diff
@@ -164,7 +164,7 @@ pub(crate) fn extract_conversion_info<'data>(
         .map_err(|_| DataError::custom("the base unit is not valid"))?;
 
     Ok(ConversionInfo {
-        basic_units: ZeroVec::from_iter(base_unit.get_single_units().iter().copied()),
+        basic_units: ZeroVec::from_iter(base_unit.get_single_units().into_iter().copied()),
```

Comment:
nit: don't call `slice.into_iter()`, it's the same as `.iter()`.
```suggestion
        basic_units: ZeroVec::from_iter(base_unit.get_single_units().iter().copied()),
```

Reply:
done